src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+
+/*
+ * Association class to support the functionality:
+ * When AddressBook exits, print out a list of all tags added/deleted during the session.
+ */
+public class Tagging {
+    private final ArrayList<String> sessionTaggings = new ArrayList<>();
```

Comment:
Recollect the discussion during tutorial on why this is not the right place to maintain the tagging list.
Please check and differentiate the 2 diagrams in the activity description for better understanding